.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Actually, lets use the webpack.config.js for this. Better hidden that way. We don't want to confuse contributors by too many top level meta files.